proto/concept.proto

```diff
@@ -0,0 +1,963 @@
+            AttributeType.GetSupertype.Res attribute_type_get_supertype_res = 402;
+            AttributeType.SetSupertype.Res attribute_type_set_supertype_res = 403;
+            AttributeType.GetRegex.Res attribute_type_get_regex_res = 407;
+            AttributeType.SetRegex.Res attribute_type_set_regex_res = 408;
+        }
+    }
+
+    message ResPart {
+        oneof res {
+            ThingType.GetOwns.ResPart thing_type_get_owns_res_part = 104;
+            ThingType.GetPlays.ResPart thing_type_get_plays_res_part = 108;
+
+            EntityType.GetSupertypes.ResPart entity_type_get_supertypes_res_part = 203;
+            EntityType.GetSubtypes.ResPart entity_type_get_subtypes_res_part = 204;
+            EntityType.GetInstances.ResPart entity_type_get_instances_res_part = 205;
+
+            RelationType.GetSupertypes.ResPart relation_type_get_supertypes_res_part = 303;
+            RelationType.GetSubtypes.ResPart relation_type_get_subtypes_res_part = 304;
+            RelationType.GetInstances.ResPart relation_type_get_instances_res_part = 305;
+            RelationType.GetRelates.ResPart relation_type_get_relates_res_part = 306;
+
+            AttributeType.GetSupertypes.ResPart attribute_type_get_supertypes_res_part = 404;
+            AttributeType.GetSubtypes.ResPart attribute_type_get_subtypes_res_part = 405;
+            AttributeType.GetInstances.ResPart attribute_type_get_instances_res_part = 406;
+            AttributeType.GetOwners.ResPart attribute_type_get_owners_res_part = 409;
+        }
+    }
+
+    message Delete {
+        message Req {}
+        message Res {}
+    }
+
+    message SetLabel {
+        message Req {
+            string label = 1;
+        }
+        message Res {}
+    }
+
+    message SetAbstract {
+        message Req {}
+        message Res {}
+    }
+
+    message UnsetAbstract {
+        message Req {}
+        message Res {}
+    }
+
+    message GetOwns {
+        message Req {
+            optional AttributeType.ValueType filter = 1;
+            Type.Transitivity transitivity = 2;
+            repeated Type.Annotation annotations = 3;
+        }
+        message ResPart {
+            repeated AttributeType attribute_types = 1;
+        }
+    }
+
+    message GetOwnsOverridden {
+        message Req {
+            AttributeType attribute_type = 1;
+        }
+        message Res {
+            optional AttributeType attribute_type = 1;
+        }
+    }
+
+    message SetOwns {
+        message Req {
+            AttributeType attribute_type = 1;
+            optional AttributeType overridden_type = 2;
+            repeated Type.Annotation annotations = 3;
+        }
+        message Res {}
+    }
+
+    message UnsetOwns {
+        message Req {
+            AttributeType attribute_type = 1;
+        }
+        message Res {}
+    }
+
+    message GetPlays {
+        message Req {
+            optional Type.Transitivity transitivity = 1;
```

Comment:
Why is transitivity optional here but not elsewhere? Lets be consistent at each place